packages/components/src/alignment-matrix-control/icon.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import { ALIGNMENTS, getAlignmentIndex } from './utils';
+import {
+	Root,
+	Cell,
+	Point,
+} from './styles/alignment-matrix-control-icon-styles';
+
+const BASE_SIZE = 24;
+
+export default function AlignmentMatrixControlIcon( {
+	className,
+	disablePointerEvents = true,
+	height,
+	size: sizeProp = BASE_SIZE,
+	style = {},
+	value = 'center',
+	width,
+	...props
```

Comment:
Feels like there's too much props for a public API component. Do we really need all these configs? I'd have thought we need just "value" and potentially "size"

Reply:
Adding a code comment re: `width` and `height`

Reply:
This would solve it right ?

```
<Icon icon={ <AlignmentMatrixControl.Icon size={ something } /> } />
```

Reply:
Indeed it would! Making those changes now